### PR TITLE
Add global navbar

### PR DESCRIPTION
**Pull request by bopke**

### CODE REVIEW

File: src/components/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    if (localStorage.getItem("jwt")) {
+        logButton = <Button color="inherit" onClick={() => {
+            history.push("/login")
+        }}>Login</Button>
+    } else {
+        logButton = <Button color="inherit" onClick={() => {
+            localStorage.clear()
+        }}>Logout</Button>
+    }
```
Comment (Graleszczur) [inline src/components/NavBar.tsx]: Good direction but creating a variable is not necessary, use {} with "?" conditional statement instead in your destined place inside component return function.
Example { localStorage.getItem("jwt") ? <Button1/> : <Button2/>}

**Review by Graleszczur — CHANGES_REQUESTED** [review]

File: src/components/NavBar.tsx (hunk 1 above)
Comment (bopke) [reply src/components/NavBar.tsx:37]: co pan mówisz ja nie rozumiem

Comment (bopke) [reply src/components/NavBar.tsx]: gosciu to nie po polsku co ty mowisz nie rozumiem

Comment (Graleszczur) [reply src/components/NavBar.tsx]: omg polack in my lab team

Comment (bopke) [reply src/components/NavBar.tsx:37]: gg surr

Comment (bopke) [reply src/components/NavBar.tsx]: gg surr 15

**Review by Graleszczur — APPROVED** [review]
GJ

**Pull request merged**